pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -104,7 +104,7 @@ func New(f kcmdutil.Factory, streams genericiooptions.IOStreams) *cobra.Command
 	flags := cmd.Flags()
 	flags.StringVar(&o.To, "to", o.To, "Specify the version to upgrade to. The version must be on the list of available updates.")
 	flags.StringVar(&o.ToImage, "to-image", o.ToImage, "Provide a release image to upgrade to.")
-	flags.BoolVar(&o.ToLatestAvailable, "to-latest", o.ToLatestAvailable, "Use the next available version.")
+	flags.BoolVar(&o.ToLatestAvailable, "to-latest", o.ToLatestAvailable, "Use the latest (largest Semantic Version) available version.")
```

Comment:
"highest" would sound more precise to me but I'm not the native english speaker here so I trust your judgement, just unhold :D

/lgtm
/hold

Reply:
I pushed "largest" -> "highest" in c04481d421fc -> 56db34e82d1c.

/hold cancel

Reply:
Here is the new output compared to the old one:
```
[jianl@jianl-thinkpadt14gen4 417]$ diff -uNb -B2 <(oc adm upgrade --help) <(./oc adm upgrade --help) | tail -n +4
 
     --to-latest=false:
-	Use the next available version.
+	Use the latest (highest Semantic Version) available version.
 
     --to-multi-arch=false:
[jianl@jianl-thinkpadt14gen4 417]$
```

I am ok with the change, a minor suggestion, maybe we just need to say `highest Semantic` in the parentheses.

Reply:
btw do we really need to capitalize the first letter of `Semantic Version`?